loadtest/artillery/tests/auth-eth/proxy.js

```diff
@@ -1,38 +1,66 @@
-import '../../../mock-browser'
+import Gun from 'gun'
+const fs = require("fs");
+const rimraf = require("rimraf");
 import bip39 from 'bip39-light'
-import Config from "../../../../src/config/config";
 import {GoodWallet} from "../../../../src/lib/wallet/GoodWalletClass";
 import {UserStorage} from "../../../../src/lib/gundb/UserStorageClass";
 import {GoodWalletLogin} from "../../../../src/lib/login/GoodWalletLogin";
-
+import { log } from '../../utils/commons'
 /**
  * Main method
  * @returns {Promise<boolean>}
  */
 const runProxy = async () => {
   try {
-   // await createCreds()
+    await generatedData()
   } catch (e) {
     console.log(e);
   }
-  
+
   return true
 }
 
+  const generatedData = async () => {
+    const count = process.env.DURATION * process.env.ARRIVALRATE
+    log('Run generated random data: ' + count)
+    let allCreds = []
+    let cred = null;
+    
+    if (fs.existsSync(`${__dirname}/temp`)) {
+      rimraf.sync(`${__dirname}/temp`);
+    }
+    fs.mkdirSync(`${__dirname}/temp`);
+    
+    for (let i=0; i < count; i++) {
+      cred = await createCreds(i)
```

Comment:
to make is quicker you should push all promises to array and then await Promise.all

Reply:
[done](https://github.com/GoodDollar/GoodDAPP/pull/561/commits/1293525771ed4004f72d85d17c92e572e0b60af6)